javascript/javascript2/week1/Week1HW.css

```
@@ -0,0 +1,12 @@
+body {
+  background-color: lightgray;
+  display: block;
+}
+.Tasks {
```

```suggestion
.tasks {
```

Never use capitalletters in html and css classes/id's